sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -67,9 +67,21 @@
    <version ClassVersion="10" checksum="2876855736"/>
   </class>
 
-  <class name="caf::SRParticleMatch" ClassVersion="10">
-   <version ClassVersion="10" checksum="3613006913"/>
-  </class>
+  <class name="caf::SRParticleMatch" ClassVersion="11">
+   <version ClassVersion="11" checksum="3613006913"/>
+   <version ClassVersion="10" checksum="3607466832"/>
+  </class>
```

Comment:
This hunk is unnecessary, right? After all this, SRParticleMatch doesn't actually change.

Reply:
ah yes, that's just me missing things when cleaning up